packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.tsx

```diff
@@ -597,4 +617,7 @@ export interface AutocompleteProps extends ComponentProps {
 	breakpoints?: BreakpointsProps;
 	controller: AutocompleteController;
 	width?: string;
+	onClickResult?: (e) => void;
```

Comment:
This should be removed in favor of using the `Result` component prop via `theme`.